cosmos/job/drm/drm_local.py

```diff
@@ -56,36 +61,58 @@ def acquire_gpus(self, task):
         self.task_id_to_gpus_used[task.id] = self.gpus_left[: task.gpu_req]
 
     def submit_job(self, task):
-        if task.time_req is not None:
-            cmd = [
-                "/usr/bin/timeout",
-                "-k",
-                "10",
-                str(task.time_req),
-                task.output_command_script_path,
-            ]
+        raise NotImplementedError("use .submit_jobs()")
+
+    def _submit_job(self, task):  # this is needed for multiprocessing
+        if self.workflow.termination_signal not in frozenset({signal.SIGINT, signal.SIGTERM, signal.SIGXCPU}):
```

Comment:
can you set a constant for frozenset({signal.SIGINT, signal.SIGTERM, signal.SIGXCPU}) in cosmos.constants.py and refer to it here, and anywhere else in the code?

call it TERMINATION_SIGNALS = frozenset({signal.SIGINT, signal.SIGTERM, signal.SIGXCPU})

Reply:
sure